test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FormatData.Tests.ps1

```diff
@@ -3,22 +3,36 @@
 Describe "Get-FormatData" -Tags "CI" {
 
     Context "Check return type of Get-FormatData" {
-
         It "Should return an object[] as the return type" {
             $result = Get-FormatData
-            ,$result | Should -BeOfType "System.Object[]"
+            , $result | Should -BeOfType "System.Object[]"
         }
     }
 
-    It "Can get format data requiring '-PowerShellVersion 5.1'" {
-        $format = Get-FormatData System.IO.FileInfo -PowerShellVersion 5.1
-        $format.TypeNames | Should -HaveCount 2
-        $format.TypeNames[0] | Should -BeExactly "System.IO.DirectoryInfo"
-        $format.TypeNames[1] | Should -BeExactly "System.IO.FileInfo"
-        $format.FormatViewDefinition | Should -HaveCount ($IsWindows ? 4 : 5)
+    Context "Local use: Can get format data requiring v5.1 by default" {
+        $cmds =
+            @{ cmd = { Get-FormatData System.IO.FileInfo } },
+            @{ cmd = { (Get-FormatData System.IO.FileInfo &) | Receive-Job -Wait -AutoRemoveJob } }
```

Comment:
Please put in `BeforeAll` block.

Reply:
Is it reliable to use `System.IO.FileInfo` format if it could be changed?

Reply:
Re `BeforeAll`: 👍

Re reliability: I agree - the point here is not to test the specifics of the format data returned (but that's how the test was originally written).

I've replaced with a simple `(& $cmd).TypeNames | Should -Contain 'System.IO.FileInfo'` - which hopefully everyone is happy with.

Reply:
As for whether to _fundamentally_ use `System.IO.FileInfo` in the tests: That type is known as one that mustn't be returned to older clients, so I think it is a reliable choice.